### DISCUSSION TITLE
❓ [QUESTION] How to perform transfer training with nequip?

### DISCUSSION THREAD

**Discussion by RA-zoL** [mir-group/nequip#245] [Q&A] [answered]
I have trained a nequip model M with a large dataset A. What can I do to make M also applicatable to a small but different dataset B? (except training a new model with A+B)

**Comment by Linux-cpp-lisp** [comment, marked as answer]
See https://github.com/mir-group/nequip/discussions/235 for discussions on how to do fine-tuning (which seems to be what you are asking?) in `nequip`.

Note that if A and B are very different this might not work very well... in general I believe this is still an open questions for machine learning atomic systems. Please let us know if you find anything interesting!

In particular, do note that B cannot contain any species that were not defined in the original model. (Technically it is possible to define a species in the original model that is not present in the original _data_ A, though I don't know how well that would work in terms of accuracy.)

**Comment by RA-zoL** [comment]
Thank you very much! That is what I want.
